homeassistant/components/matter/select.py

```diff
@@ -436,4 +436,27 @@ def _update_from_device(self) -> None:
         # don't discover this entry if the supported rinses list is empty
         secondary_value_is_not=[],
     ),
+    MatterDiscoverySchema(
+        platform=Platform.SELECT,
+        entity_description=MatterSelectEntityDescription(
+            key="DoorLockSoundVolume",
+            entity_category=EntityCategory.CONFIG,
+            translation_key="door_lock_sound_volume",
+            options=["silent", "low", "medium", "high"],
+            measurement_to_ha={
+                0: "silent",
+                1: "low",
+                3: "medium",
+                4: "high",
+            }.get,
+            ha_to_native_value={
+                "silent": 0,
+                "low": 1,
+                "medium": 3,
+                "high": 2,
```

Comment:
This should probably correspond to line 450, right? Either change here or there.
```suggestion
                "high": 4,
```

Reply:
Line 450 was the mistake - Matter App. Cluster section 5.2.6.23 says the "high" value is 2. Thanks for catching this.